### PR TITLE
pkg/utils: add utils_test

### PR DESCRIPTION
**Pull request by q384566678**
Add corresponding unit tests to the functions in util.go.

Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: pkg/utils/utils_test.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+		{
+			description: "normal containerID",
+			containerID: "a51debf7e1eb",
+			err:         false,
+		},
+	}
+
+	for _, tt := range testData {
+		err := ValidateContainerID(tt.containerID)
+		if err != nil && !tt.err {
```
Comment (mars1024) [inline pkg/utils/utils_test.go]: Could you help to use DeepEqual to squash these lines?

**Comment by mars1024** [comment]
BTW, commits need to be squashed either.

File: pkg/utils/utils_test.go (hunk 1 above)
Comment (q384566678) [reply pkg/utils/utils_test.go]: They are different structure types, and it seems that it is not suitable to use DeepEqual. If they are changed to agree types and then compared, more code will be generated.

Comment (mars1024) [reply pkg/utils/utils_test.go]: FYI https://play.golang.org/p/DicOkjUajmv

Comment (q384566678) [reply pkg/utils/utils_test.go]: updated, thanks for your suggestions. PTAL

**Pull request merged**